arctic/store/_ndarray_store.py

```diff
@@ -145,9 +143,6 @@ def _index_range(self, version, symbol, from_version=None, **kwargs):
         """
         from_index = None
         if from_version:
-            if version['base_sha'] != from_version['base_sha']:
```

Comment:
per a conversation with Richard:

As you found, base_sha is used as a consistency check when ‘tailing’ the contents of a time series – we would read only the difference between one version and a subsequent version, assuming that there had only been appends between versions. We removed that functionality, so base_sha can probably be deleted.